### PR TITLE
chore: Test "jx get build logs" on completed builds in Lighthouse test

### PR DESCRIPTION
**Pull request by abayer**
Signed-off-by: Andrew Bayer <andrew.bayer@gmail.com>

### CODE REVIEW

**Comment by abayer** [comment]
This works - the LH PR hasn't had everything pass yet, but that's because it needs https://github.com/jenkins-x/jx/pull/7218. =)

**Comment by abayer** [comment]
/hold

Holding until I've actually got the fixes working correctly.

File: test/helpers/test.go [hunk 1]
```diff
@@ -810,15 +810,24 @@ func isADesiredStatus(status string, desiredStatuses []string) bool {
 	return false
 }
 
-// TailBuildLog tails the logs of the specified job
-func (t *TestOptions) TailBuildLog(jobName string, maxDuration time.Duration) {
+// TailSpecificBuildLog tails the logs of the specified job and number, not passing a specific build number to "jx get build logs"
+// if the build number is 0.
+func (t *TestOptions) TailSpecificBuildLog(jobName string, buildNumber int, maxDuration time.Duration) {
```
Comment (hferentschik) [inline test/helpers/test.go:815]: :-)

File: test/helpers/test.go [hunk 2]
```diff
@@ -810,15 +810,24 @@ func isADesiredStatus(status string, desiredStatuses []string) bool {
...
+		args = append(args, "--build", strconv.Itoa(buildNumber))
+	}
 	argsStr := strings.Join(args, " ")
 	By(fmt.Sprintf("checking that there is a job built successfully by calling jx %s", argsStr), func() {
 		t.ExpectJxExecution(t.WorkDir, maxDuration, 0, args...)
 	})
 }
 
+// TailBuildLog tails the logs of the specified job, getting the latest build.
+func (t *TestOptions) TailBuildLog(jobName string, maxDuration time.Duration) {
```
Comment (hferentschik) [inline test/helpers/test.go:827]: 👍

**Pull request merged**